console/src/components/Common/FilterQuery/FilterQuery.tsx

```diff
@@ -1,41 +1,55 @@
 import React from 'react';
+import { connect, MapStateToProps, ConnectedProps } from 'react-redux';
+import { RouteComponentProps, withRouter } from 'react-router';
+import { compose } from 'redux';
+
 import { OrderBy } from '../utils/v1QueryUtils';
 import Where from './Where';
 import Sorts from './Sorts';
 import { useFilterQuery } from './state';
 import { Filter, FilterRenderProp } from './types';
-import { Dispatch } from '../../../types';
+import { ReduxState } from '../../../types';
 import ReloadEnumValuesButton from '../../Services/Data/Common/Components/ReloadEnumValuesButton';
 import Button from '../Button/Button';
 import { Nullable } from '../utils/tsUtils';
 import styles from './FilterQuery.scss';
 import { BaseTable } from '../../../dataSources/types';
 import { generateTableDef } from '../../../dataSources';
+import { mapDispatchToPropsEmpty } from '../utils/reactUtils';
 
-type Props = {
+interface Props extends InjectedProps {
```

Comment:
@beerose the props here are incorrect. This is the error that is reported: ` JSX element type 'FilterQuery' does not have any construct or call signatures.`